src/app/components/challenge/challenge.component.ts

```diff
@@ -325,4 +325,25 @@ export class ChallengeComponent implements OnInit {
     this.globalService.showConfirm(PARAMS);
   }
 
+  inviteUsertoChallengeWrapper() {
+    const SELF = this;
+    const inviteUsertoChallenge = (e) => {
+      SELF.apiCall = (params) => {
+        const BODY = JSON.stringify(params);
+        SELF.apiService.postUrl('challenge/' + this.challenge.id + '/send-invitation/', BODY).subscribe(
+          data => {
+            SELF.globalService.showToast('success', 'User invited for the challenge successfully', 5);
```

Comment:
@Kajol-Kumari Why you added this(^) function in this PR?

Reply:
Left by mistake.will remove it.Thank you for pointing out.